conftest.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bats
+
+@test "/project/image/policy/deprecated-labels" {
+  run conftest test --policy $POLICY_PATH/image/policy/deprecated-labels.rego $JSON_PATH
+  [ "$status" -eq 0 ]
+}
+
+@test "/project/image/policy/required-labels" {
+  run conftest test --policy $POLICY_PATH/image/policy/required-labels.rego $JSON_PATH
+  [ "$status" -eq 0 ]
+}
+
+@test "/project/image/policy/deprecated-image" {
+  run conftest test --policy $POLICY_PATH/image/policy/deprecated-image.rego $JSON_PATH
```

Comment:
The deprecated-image policy is meant to be executed on the Pyxis repositories data, e.g. a repository that's supposed to pass: https://catalog.redhat.com/api/containers/v1/repositories/registry/registry.access.redhat.com/repository/ubi8/ubi and one that's supposed to fail the test: https://catalog.redhat.com/api/containers/v1/repositories/registry/registry.access.redhat.com/repository/rhscl/nodejs-8-rhel7

Reply:
The latest deprecated image policy in `policies/repository/deprecated-image.rego` getting passed for both the pyxis repositories.